pkg/ha/state/state.go

```diff
@@ -1,23 +1,41 @@
 package state
```

Comment:
should this file be in pkg/ha/state/lease.go or  pkg/ha/lease/lease.go?

Reply:
will leave it as state/lease so we don't end up with an import of lease.Lease